perception/image_projection_based_fusion/launch/pointpainting_fusion.launch.xml

```diff
@@ -18,7 +18,7 @@
   <arg name="input/camera_info7" default="/camera_info7"/>
   <arg name="input/pointcloud" default="/sensing/lidar/top/rectified/pointcloud"/>
   <arg name="output/objects" default="objects"/>
-  <arg name="model_name" default="pointpainting" description="options: `pointpainting`"/>
+  <arg name="model_name" default="waymo" description="options: `waymo`"/>
```

Comment:
What reason did you change from pointpainting to waymo?

Reply:
I wanted to give a different name to the multi-frame model.

Reply:
It may not be a good to mention the company name. can you change waymo_open_dataset?

Reply:
Thanks, I changed to waymo_open_dataset.